integration-test/run-it.sh

```diff
@@ -101,3 +104,7 @@ for i in {1..20} ; do
 done
 list_storm_processes
 mvn test -DfailIfNoTests=false -DskipTests=false -Dstorm.version=${STORM_VERSION} -Dui.url=http://localhost:8744
+
+# Kill all storm processes after tests running, otherwise the Travis CI(ARM) will still wait and then fail by
+# timeout error even though all the tests passed
+sudo pkill -9 -u storm
```

Comment:
Is this change still needed after the change to arm64 and xenial

Reply:
Saw this in your test job  https://travis-ci.com/github/apache/storm/jobs/460903786
```
No output has been received in the last 10m0s, this potentially indicates a stalled build or something wrong with the build itself.
Check the details on how to adjust your build configuration on: https://docs.travis-ci.com/user/common-build-problems/#build-times-out-because-no-output-was-received
The build has been terminated
```
Is this what you saw before?

Reply:
yes, I have tried again in #3361, you can take a look the CI logs: https://travis-ci.com/github/apache/storm/jobs/460903786
at the end of the log, even all the intgration tests ran sucessfully, but the job didn't exit immediately and then failed due to timeout. 
```
0.11s$ rm -rf $HOME/.m2/repository/org/apache/storm
cache.2
store build cache
Done. Your build exited with 0.
No output has been received in the last 10m0s, this potentially indicates a stalled build or something wrong with the build itself.
Check the details on how to adjust your build configuration on: https://docs.travis-ci.com/user/common-build-problems/#build-times-out-because-no-output-was-received
The build has been terminated

```

Reply:
Thanks